src/utils/api.ts

```diff
@@ -0,0 +1,33 @@
+import { PaginationOrder } from "@hermeznetwork/hermezjs/src/api";
+
+// domain
+import { HermezApiResourceItem } from "src/domain/hermez";
+
+export interface Pagination {
+  fromItem?: number;
+  hasMoreItems: boolean;
+}
```

Comment:
If `fromItem` is required when `hasMoreItems` is true this would be better modeled as a union.

Reply:
I agree here! `fromItem` should only be there if `hasMoreItems === true`.

Reply:
Ok it's now modeled as a union.